app/views/monitoring.html

```diff
@@ -303,7 +303,9 @@
                               <small>created <span am-time-ago="build.metadata.creationTimestamp"></span></small>
                             </div>
                             <div class="list-group-item-text">
-                              <build-status build="build"></build-status>
```

Comment:
Should we delete the build-status directive if it's not used? Or update it to show the reason or message instead?